crates/testlang/outputs/cargo/crate/Cargo.toml

```diff
@@ -0,0 +1,54 @@
+[package]
+version.workspace = true
```

Comment:
I also suggest removing all the additional metadata, as they are only used for public facing artifacts (across languages), and are maintained together: description, URLS, contributors, keywords, etc...